projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -115,6 +115,25 @@ export interface FeatureTogglesInterface {
    */
   productConfiguratorAttributeTypesV2?: boolean;
 
+  /**
+   * In a server environment (SSR or Prerendering) it propagates all errors caught in Angular app
+   * (in the Angular's `ErrorHandler` class) to the server layer.
+   *
+   * In SSR, such a propagation allows the server layer (e.g. ExpressJS) for handling those errors,
+   * e.g. sending a proper Error Page in response to the client,
+   * instead of a rendered HTML that is possibly malformed due to the occurred error.
+   */
+  propagateErrorsToServer?: boolean;
+
+  /**
+   * In SSR, the following errors will be printed to logs (and additionally can also
+   * be forwarded to ExpressJS if only the setting
+   * `SsrOptimizationOptions.ssrErrorHandling`  is enabled):
```

Comment:
[MEDIUM]
Is this JSDoc outdated?
```suggestion
   * be forwarded to ExpressJS if only the other feature toggle
   * `propagateErrorsToServer` is enabled):
```